kolibri/core/settings.py

```diff
@@ -0,0 +1,27 @@
+"""
+Kolibri core settings module
+============================
+
+This module is *not* the project settings but defines variables that can be
+configured through the project settings - and their defaults.
+
+If something in the core uses a custom project setting, it should use a
+naming scheme in django.conf.settings like ``KOLIBRI_*``.
+
+Please avoid depending on settings in kolibri.deployments.default.settings,
+rather define the default here so we don't have multiple layers of defaults.
+"""
+
+from django.conf import settings
+
+# : These plugins are automatically enabled if they exist at the first run of
+# : Kolibri. After that, they are not searched for automatically since that would
+# : require explicitly disabling them in case they were to be removed on purpose
+# : from INSTALLED_APPS.
+WELL_KNOWN_PLUGINS = getattr(
+    settings,
+    'KOLIBRI_WELL_KNOWN_PLUGINS',
+    [
+        'kolibri.plugins.example_plugin',
+    ]
+)
```

Comment:
I think I would still prefer the name `DEFAULT_PLUGINS` here - because if they exist, we activate them by default. `WELL_KNOWN` doesn't really indicate that we will activate them.